ddp-workspace/projects/ddp-testboston/src/assets/i18n/en.json

```diff
@@ -231,7 +231,8 @@
             "Summary": "Summary",
             "ActivityStatus": "Status",
             "ActivityDate": "Created",
-            "ActivityActions": "Actions"
+            "ActivityActions": "Actions",
+            "ActivityQuestionCount": "{{questionsAnswered}} of {{questionTotal}} Answered"
         },
         "ManageParticipants": {
```

Comment:
The translation. Since by default our configuration does not show question counts we only have to set it up for testboston.